### DISCUSSION TITLE
PINNs energy method

### DISCUSSION THREAD

**Discussion by HumbertHumbert7** [lululxvi/deepxde#1713] [Q&A]
I’m trying to use deepxde for pinns in the variational formulation. I started with a veruly trivial case: the potential energy of a beam under a distributed load that is just the integral between 0 and 1 of w_xx**2 minus the integral of q*w, where w is the unknown displacement and q the distributed load. I have computed the integral with tf.reduce_sum(w_xx**2-q*x,axis=1) and I have use in the pde definition, instead of the strong form. The problem is that I always obtain zero displacement. How can I avoid this?
